plugins/uefi/fu-uefi-tool.c

```diff
@@ -305,6 +310,28 @@ main (int argc, char *argv[])
 		g_print ("%s\n", _("Disabled fwupdate debugging"));
 	}
 
+	if (action_pcr) {
```

Comment:
I think fwupdate should really be a deprecated command. I think we can do a much smaller binary that's also suitable for fuzz testing.